src/library/scala/util/control/Exception.scala

```diff
@@ -66,8 +66,8 @@ import scala.language.implicitConversions
  *
  *  val goodUrl = "http://www.scala-lang.org/"
  *  // Prints page content,
- *  //   &lt;!DOCTYPE html&gt;
- *  //   &lt;html&gt;
+ *  //   <!DOCTYPE html>
+ *  //   <html>
```

Comment:
Seems this is a breaking change for existing Scaladoc comments.  What's the consequences for users (existing Scaladoc code)?

Reply:
Good piont! We could identify existing html escapes and avoid escaping `&` to `&amp;` in this case. @xuwei-k

Reply:
fixed